### PR TITLE
User can view full recipe

### PR DESCRIPTION
**Pull request by beccaburns**
[PT-Link](https://www.pivotaltracker.com/story/show/168979795)

## In this PR
- Created a function for the FoodService module to allow user to view full recipe with image (show action included within the controller for this function)
- Updated styling for the website layout

## What we learned
- How to pull different attributes from the API depending on what we want to showcase
- How to stub (still a bit unclear for us at this time)
- Continued learning of HAML and CSS

Screen Shot 2019-10-10 at 1 08 16 PM

### CODE REVIEW

**Review by piavW — APPROVED** [review]
Please resolve merge conflicts.

**Review by yasmineezequiel — APPROVED** [review]

File: config/routes.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 Rails.application.routes.draw do
   root controller: :recipes, action: :index
-  resources :recipes, only: [:index]
+  resources :recipes
```
Comment (faraznaeem) [inline config/routes.rb]: please restrict the actions

File: features/user_can_view_full_recipe.feature [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+Feature: User can view full recipe
+  As a user
+  In order to see a recipe
+  I would like to be able to press a button called show to see the full recipe
+
+  Scenario: User views full recipe with instructions
+    Given I visit the home page
+    When I fill in "Search" with "apples"
+    And I click "Submit"
+    Then I click "Dried Apples"
```
Comment (faraznaeem) [inline features/user_can_view_full_recipe.feature]: `And`

File: features/user_can_view_full_recipe.feature [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+    And I should see "Position racks in the upper and lower third of the oven; preheat to 200F. Line 2 large baking sheets with parchment paper."
```
Comment (faraznaeem) [inline features/user_can_view_full_recipe.feature]: `Then`

**Review by faraznaeem — CHANGES_REQUESTED** [review]

File: features/user_can_view_full_recipe.feature (hunk 2 above)
Comment (beccaburns) [reply features/user_can_view_full_recipe.feature]: @faraznaeem Oliver told us to type Then and And opposite of what you are advising. Perhaps you can help us understand why you choose to do it this way instead?

Comment (faraznaeem) [reply features/user_can_view_full_recipe.feature]: Beacuse the assertion is the the next step `And I should see`, the `Then` is a continuation of the scenario. 
I leave it up to you to decide what you want to keep.

Comment (beccaburns) [reply features/user_can_view_full_recipe.feature]: Yes, that makes sense to us. We were just getting mixed information.

Comment (faraznaeem) [reply features/user_can_view_full_recipe.feature]: No worries, change it then it is time for a merge

**Pull request merged**